ghga_transpiler/cli.py

```diff
@@ -20,20 +20,36 @@
 
 import typer
 
-HERE = Path(__file__).parent.resolve()
-DEFAULT_OUTPUT_FILE = HERE / "transpiled_metadata.yaml"
+from .config import load_config
+from .config.exceptions import MissingWorkbookContent
+from .core import convert_rows, get_header, get_worksheet_rows, read_workbook
 
 cli = typer.Typer()
 
+CONFIG = load_config("0.0.1")
 
-def convert_workbook(filename):
-    """Function to run steps for conversion
 
-    This is a place holder
-
-    """
+def convert_workbook(filename: Path):
+    """Function to convert an input spreadsheet into JSON"""
+    converted_workbook = {}
+    workbook = read_workbook(str(filename))
+    for sheet in CONFIG.worksheets:
+        assert sheet.settings is not None  # nosec
+        try:
+            rows = get_worksheet_rows(
+                workbook[sheet.sheet_name],
+                sheet.settings.start_row,
+                workbook[sheet.sheet_name].max_row,
+                sheet.settings.start_column,
+                sheet.settings.end_column,
+            )
+        except KeyError as exc:
+            raise MissingWorkbookContent(
+                f"Workbook does not contain {sheet.sheet_name} worksheet."
+            ) from exc
```

Comment:
```suggestion
            )
```
You are catching an error that is based on invalid user input. We know the cause, providing the user with the exception trace is not helpful. In fact, this exception should be caught in the cli command and transformed into a regular CLI error message.

Reply:
Pylint insists on[ `as exc` `from exc`](https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/raise-missing-from.html)